spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/MultipleJGitEnvironmentRepository.java

```diff
@@ -160,7 +160,6 @@ public Environment findOne(String application, String profile, String label, boo
 							this.logger.debug("Cannot load configuration from " + candidate.getUri() + ", cause: ("
 									+ e.getClass().getSimpleName() + ") " + e.getMessage(), e);
 						}
-						continue;
```

Comment:
Why is this being removed?

Reply:
My IDE suggests
```
'continue' is unnecessary as the last statement in a loop 
```

so i removed it.
I find that there is one more such continue statement at `MultipleJGitEnvironmentRepository.getLocations`

May i remove 'continue;' from that also or retreive this one.

Reply:
please limit changes to those specific to the issue.